src/sqladmin-fetcher.ts

```diff
@@ -146,10 +146,31 @@ export class SQLAdminFetcher {
       });
     }
 
-    const ipAddresses = parseIpAddresses(
-      res.data.ipAddresses,
-      res.data.dnsName
-    );
+    const ipAddresses: IpAddresses = {};
```

Comment:
I liked this better when it was it's own function, `parseIpAddress()`. Is there a reason you decided to move the logic inline here?